test/config/postfix-regexp.cf

```diff
@@ -1,2 +1,3 @@
 /^test[0-9][0-9]*@localhost.localdomain/ user1@localhost.localdomain
 /^bounce.*@.*/ external1@otherdomain.tld
+/^postmaster@/ user1@localhost.localdomain
```

Comment:
Can you add a new line.

Reply:
as I mentioned in the PR body, the tests are failing before my changes as the counts are already wrong - I assume some earlier change broke these tests.